ray_ci/build_base_build.sh

```diff
@@ -1,89 +1,98 @@
 #!/bin/bash
-set -e
 
-export DOCKER_BUILDKIT=1
+set -euo pipefail
 
-cd "$RAY_REPO_DIR" || true
+export DOCKER_BUILDKIT=1
 
-if [ "$BUILDKITE_COMMIT" = "HEAD" ]; then
-  export BUILDKITE_COMMIT=$(git log -1 --format="%H")
-  echo "Resolved BUILDKITE_COMMIT to $BUILDKITE_COMMIT"
+if [[ "${RAY_REPO_DIR:-}" != "" ]]; then
```

Comment:
is this a new env introduced in this PR? does it matter?

Reply:
it was referenced on line 6? I just trying to make it run with `set -u` even when it is running locally.

this var is set to specify where the ray repo is checked out.

this script is a legacy script that is being used today on civ1. I am doing the refactoring to make it easier to co-exist with civ2.